full/full_coupler_mod.F90

```diff
@@ -1332,7 +1343,7 @@ end subroutine add_domain_dimension_data
 
   !> \brief Writing restart file that contains running time and restart file writing time.
   subroutine coupler_restart(Atm, Ice, Ocean, Ocn_bc_restart, Ice_bc_restart, &
-                             Time_run, Time_res, Time_start, Time_end, time_stamp)
+                            Time_current, Time_restart_current, Time_start, time_stamp)
```

Comment:
I don’t like changing the interface for a high-level routine like this.  Is this part of the refactor plan?

Reply:
This is a new subroutine part of the rewrite, so it's not merged into any existing release.